WordPress/Classes/ViewRelated/Post/PostMenuHelper.swift

```diff
@@ -121,25 +121,25 @@ extension PostCardStatusViewModel.Button: PostMenuAction {
             delegate.cancelAutoUpload(post)
         case .share:
             delegate.share(post, fromView: view)
-        case .copyLink:
-            delegate.copyLink(post)
         case .blaze:
             delegate.blaze(post)
+        case .comments:
+            delegate.comments(post)
         }
     }
 
     private enum Strings {
-        static let cancelAutoUpload = NSLocalizedString("posts.cancelUpload.actionTitle", value: "Cancel Upload", comment: "Label for the Post List option that cancels automatic uploading of a post.")
-        static let stats = NSLocalizedString("posts.stats.actionTitle", value: "Stats", comment: "Label for post stats option. Tapping displays statistics for a post.")
+        static let cancelAutoUpload = NSLocalizedString("posts.cancelUpload.actionTitle", value: "Cancel upload", comment: "Label for the Post List option that cancels automatic uploading of a post.")
+        static let stats = NSLocalizedString("posts.stats.actionTitle", value: "Go to stats", comment: "Label for post stats option. Tapping displays statistics for a post.")
+        static let comments = NSLocalizedString("posts.comments.actionTitle", value: "Go to comments", comment: "Label for post comments option. Tapping comments statistics for a post.")
         static let duplicate = NSLocalizedString("posts.duplicate.actionTitle", value: "Duplicate", comment: "Label for post duplicate option. Tapping creates a copy of the post.")
-        static let publish = NSLocalizedString("posts.publish.actionTitle", value: "Publish Now", comment: "Label for an option that moves a publishes a post immediately")
-        static let draft = NSLocalizedString("posts.draft.actionTitle", value: "Move to Draft", comment: "Label for an option that moves a post to the draft folder")
-        static let delete = NSLocalizedString("posts.delete.actionTitle", value: "Delete Permanently", comment: "Label for the delete post option. Tapping permanently deletes a post.")
-        static let trash = NSLocalizedString("posts.trash.actionTitle", value: "Move to Trash", comment: "Label for a option that moves a post to the trash folder")
+        static let publish = NSLocalizedString("posts.publish.actionTitle", value: "Publish now", comment: "Label for an option that moves a publishes a post immediately")
+        static let draft = NSLocalizedString("posts.draft.actionTitle", value: "Move to draft", comment: "Label for an option that moves a post to the draft folder")
+        static let delete = NSLocalizedString("posts.delete.actionTitle", value: "Delete permanently", comment: "Label for the delete post option. Tapping permanently deletes a post.")
```

Comment:
We may need a separate US version of the localization. These need to be capital-cased in the US, as in many other places in the app that are not capitalizing the titles.